common/api/endpoint/ApiEndPoint.ts

```diff
@@ -43,6 +43,10 @@ export const ApiEndPoint = {
     OriginalDataPoint: {
       one: (countryIso = ':countryIso', assessmentName = ':assessmentName', cycleName = ':cycleName', year = ':year') =>
         apiPath('country', countryIso, 'assessment', assessmentName, 'originalDataPoint', cycleName, year),
+
+      many: (countryIso = ':countryIso', assessmentName = ':assessmentName', cycleName = ':cycleName') =>
```

Comment:
Done with old convention to be in line with previous (`one`) for easier refactor later